### DISCUSSION TITLE
Whether the files in the src/generated/linux directory are written manually or automatically generated by some tool

### DISCUSSION THREAD

**Discussion by kkbugnj** [microsoft/msquic#3650] [Q&A] [answered]
Whether the files in the src/generated/linux directory are written manually or automatically generated by some tool?

**Comment by nibanks** [comment, marked as answer]
I think those are the files generated by our usage of [CLOG](https://github.com/microsoft/CLOG).

**Reply by kkbugnj** [reply]
thanks a lot
